collections/src/main/scala/strawman/collection/package.scala

```diff
@@ -121,8 +121,8 @@ package object collection extends LowPriority {
 
   /** An extractor used to init/last deconstruct sequences. */
   object :+ {
-    /** Splits a sequence into init :+ last.
-      * @return Some((init, last)) if sequence is non-empty. None otherwise.
+    /** Splits a sequence into init :+ tail.
+      * @return Some((init, tail)) if sequence is non-empty. None otherwise.
```

Comment:
This is wrong. `:+` does split the sequence into `(init, last)`.

Reply:
Oops, indeed…